src/components/Modals/ContactFormModal.spec.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+
+import { DumbContactFormModal, getUpdatedContact } from './ContactFormModal'
+
+Date.now = jest.fn(() => 1548860812000)
+
+describe('ContactFormModal component', () => {
+  describe('getUpdatedContact function', () => {
+    it('should get params to create a contact if it does not exist yet', async () => {
```

Comment:
Nit: I don't really understand the name of the test, what does "get params" refer to? Why not something like "it should add cozy metadata to a new contact"?

Reply:
:ok_hand: